src/java.base/share/classes/java/util/concurrent/FutureTask.java

```diff
@@ -493,25 +493,19 @@ else if (!WAITERS.compareAndSet(this, q, s))
      * @return a string representation of this FutureTask
      */
     public String toString() {
```

Comment:
Classes in java.util.concurrent are handled upstream. It would probably be better to leave them out of this patch. Or synchronize with @DougLea to see how to bring these changes in the upstream repo.

Reply:
I rolled back changes in java.util.concurrent. Also, rebased the branch due to conflicts in JapaneseImperialCalendar.java